### ISSUE TITLE
Mail to Gmail spam box

### ISSUE THREAD

**Issue by anewjump** [PHPMailer/PHPMailer#907]
I'm using PHPMailer 5.2.14. The message was sent to gmail but every time the message shown in spam box. I had already set SPF record correctly.
When I tried using direct sendmail using telnet 25 command with same server and  same 'mail from' and same 'rcpt to', the email was delivered to in-box correctly.
Does anyone have idea?

--- SPAM BOX mail msg created by PHPMailer -----
```
Delivered-To: anewjump@gmail.com
Received: by 10.50.179.9 with SMTP id dc9csp1853182igc;
        Tue, 20 Dec 2016 19:05:13 -0800 (PST)
X-Received: by 10.99.48.68 with SMTP id w65mr4125697pgw.68.1482289513853;
        Tue, 20 Dec 2016 19:05:13 -0800 (PST)
Return-Path: <letdeal@letdeal.com>
Received: from Quick126-120.localdomain ([103.60.126.120])
        by mx.google.com with ESMTP id k79si24792396pfj.2.2016.12.20.19.05.13
        for <anewjump@gmail.com>;
        Tue, 20 Dec 2016 19:05:13 -0800 (PST)
Received-SPF: pass (google.com: domain of letdeal@letdeal.com designates 103.60.126.120 as permitted sender) client-ip=103.60.126.120;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of letdeal@letdeal.com designates 103.60.126.120 as permitted sender) smtp.mailfrom=letdeal@letdeal.com
Received: from letdeal.com (localhost [127.0.0.1]) by Quick126-120.localdomain (Postfix) with ESMTP id 76D8BE02BB for <anewjump@gmail.com>; Wed, 21 Dec 2016 12:05:23 +0900 (KST)
Date: Wed, 21 Dec 2016 12:05:23 +0900
To: anewjump@gmail.com
From: "LETDEAL.com" <letdeal@letdeal.com>
Subject: [렛딜닷컴]회원가입을 환영합니다
Message-ID: <792f016d619d22bf876cb2b9f4cc4722@letdeal.com>
X-Mailer: PHPMailer 5.2.14 (https://github.com/PHPMailer/PHPMailer)
MIME-Version: 1.0
Content-Type: multipart/alternative; boundary="b1_792f016d619d22bf876cb2b9f4cc4722"

--b1_792f016d619d22bf876cb2b9f4cc4722
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: base64

6rCE7Y64IO2ajOybkOqwgOyehSDsmYTro4wg66Cb65Sc64u37Lu0IO2ajOybkOqwgOyeheydhCDq
sJDsgqzrk5zrpr3ri4jri6Qu6rCA7J6F7ZWY7IugIOygleuztOuhnCDslYjsoITtlZjqsowg66Gc
6re47J247ZWY7IukIOyImCDsnojsirXri4jri6QuDQoJDQoNCg0KCe2ajOybkOuLmOydmCDqsIDs
```

--------- Tested sendmail direct using telnet -----------------------
(even if there was no subject, it was delivered correctly to In-box)
```
Delivered-To: anewjump@gmail.com
Received: by 10.50.179.9 with SMTP id dc9csp1455488igc;
        Tue, 20 Dec 2016 02:43:46 -0800 (PST)
X-Received: by 10.98.70.150 with SMTP id o22mr18549142pfi.134.1482230626180;
        Tue, 20 Dec 2016 02:43:46 -0800 (PST)
Return-Path: <letdeal@letdeal.com>
Received: from Quick126-120.localdomain ([103.60.126.120])
        by mx.google.com with ESMTP id b21si21644204pgh.277.2016.12.20.02.43.45
        for <anewjump@gmail.com>;
        Tue, 20 Dec 2016 02:43:45 -0800 (PST)
Received-SPF: neutral (google.com: 103.60.126.120 is neither permitted nor denied by best guess record for domain of letdeal@letdeal.com) client-ip=103.60.126.120;
Authentication-Results: mx.google.com;
       spf=neutral (google.com: 103.60.126.120 is neither permitted nor denied by best guess record for domain of letdeal@letdeal.com) smtp.mailfrom=letdeal@letdeal.com
Received: from localhost (localhost [IPv6:::1]) by Quick126-120.localdomain (Postfix) with SMTP id 2FDDBE02B2 for <anewjump@gmail.com>; Tue, 20 Dec 2016 19:43:13 +0900 (KST)
Message-Id: <20161220104341.2FDDBE02B2@Quick126-120.localdomain>
Date: Tue, 20 Dec 2016 19:43:13 +0900 (KST)
From: letdeal@letdeal.com
To: undisclosed-recipients:;

test from dcpark
hihihihihih
```

**Comment by Synchro** [comment]
This makes little sense:

    Received-SPF: neutral (google.com: 103.60.126.120 is neither permitted nor denied by best guess record for domain of letdeal@letdeal.com) client-ip=103.60.126.120;

vs.

    Received-SPF: pass (google.com: domain of letdeal@letdeal.com designates 103.60.126.120 as permitted sender) client-ip=103.60.126.120;

You've got the same IP giving different SPF results for the same domain. I can only conclude you changed your SPF record between these messages.

Fundamentally, you have no control over inbox placement; spam filtering is entirely arbitrary and controlled by the receiver. SPF does not make any guarantees at all about whether your message is treated as spam or not - it only authenticates that the source is legitimate for the domain, which can be just as true for spam as for non-spam.

In your first message, your subject line is not 7-bit clean; PHPMailer does not support SMTPUTF8, so that's unlikely to work, and I'm curious how it ended up like that in the delivered message.

I also suggest updating to latest PHPMailer - you're using an old version.

**Comment by anewjump** [comment]
First of all, Thank you for your quick reply.
Yes, you are correct the change of SPF records was made between fiirst and second.
By the way, I think Gmail server don't much care about SPF when judge spam or not. 
And I want to tell you I had gave up using local SMTP send mail. Actually, @letdeal.com has another commercial SMTP server by email hosting company. So I had change my way to send using that server on the setting and succeeded.

So, finally I knew that there is nothing relation spam or not on PHPMailer's setting.

As you know, I had set

```
$mail->Host = "smtp.xxxxx.com"; // SMTP server
$mail->SMTPAuth = true;
$mail->Username   = "letdeal@letdeal.com";
$mail->Password   = "xxxxxxx";
```

And messege had sent to In-box correctly.
Thank you for your advice of all, especially your comment about subject (not 7-bit clean), But I had gave up because I had so tired to try. And as you metioned, we cannot guarantee all about different spam polishs. Let it concern by commercial email services company.

Thanks a lot Synchro
